plugins/circleci/test/circleci-config.test.ts

```diff
@@ -18,10 +18,19 @@ jest.mock('fs', () => {
 const mockedWriteFile = jest.mocked(fs.promises.writeFile)
 
 describe('CircleCI config hook', () => {
-  class TestHook extends CircleCiConfigHook {
+  abstract class FakeCircleCiConfigHook extends CircleCiConfigHook {
+    async getCircleConfig(): Promise<[string | undefined, boolean]> {
+      const [config] = await super.getCircleConfig()
+      // pretend it's never the first time we've opened the config so that we
+      // skip the comparison with the base config generated by
+      // getInitialState()
+      return [config, false]
```

Comment:
**chore:** We should add a test for when it has been the config has been checked once before.